packages/relayer/indexer/indexer.go

```diff
@@ -95,6 +95,8 @@ type Indexer struct {
 
 	numLatestBlocksToIgnoreWhenCrawling uint64
 
+	targetBlock *uint64
+
```

Comment:
this should may be called "targetBlockNumber"

Reply:
Updated.